runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1661,7 +1661,7 @@ func s:CreateBreakpoint(id, subid, enabled)
       endif
     endif
     call sign_define('debugBreakpoint' .. nr,
-				\ #{text: strpart(label, 0, 2), 
+				\ #{text: (strchars(label) > 2 ? strpart(label, 0, 2) : label),
```

Comment:
It seems better to use `strcharpart()` or `slice()` instead of `strpart()`.

Reply:
```suggestion
				\ #{text: label->slice(0, 2),
```

Reply:
maybe can check with strdisplaywidth() tho not necessary, but slice() seems not directly meaning sometime.
@zeertzjq what do you think?